isisd/isisd.c

```diff
@@ -77,7 +77,6 @@ unsigned long debug_sr;
 
 struct isis *isis = NULL;
 
-DEFINE_QOBJ_TYPE(isis)
```

Comment:
Do we need to have this for the ISIS-VRF work going on?

Reply:
isisd shouldn't need it since the daemon was already converted to YANG (so we use `VTY_PUSH_XPATH` instead of `VTY_PUSH_CONTEXT`, for example). fabricd, however, might need this code for VRF support since it wasn't converted to YANG yet.
Since Kaushik isn't planning to add VRF support to fabricd now, I think it's fine to remove this. It's probably better to convert fabricd to YANG before doing any further work on it.